Framework/DataHandling/src/LoadNXSPE.cpp

```diff
@@ -338,7 +338,11 @@ void LoadNXSPE::exec() {
       }
     }
   }
-
+  // For NXSPE files generated by Mantid using the usual direct reduction
+  // on ISIS instruments, the data is actually a distribution.
+  if (instrument_name == "LET" || instrument_name == "MAPS" ||
```

Comment:
I don't know. I think so, but I'm not sure about SNS instruments. I'll ask Andrei...

Reply:
Indirect also use SofQW!

Reply:
@SpencerHowells Duc's changes for SofQW are purely for speed. This change only relates to loading `.nxspe` files that you guys don't use.

Reply:
@martyngigg @SpencerHowells @AndreiSavici So I've checked the workspaces which are saved as `NXSPE` by the reduction routines and system tests that for all instruments (SNS and ISIS, direct and indirect) that the workspaces are all `distribution` when saved. However `SaveNXSPE` then discards this because the `NXSPE` format itself doesn't have this flag. So I think we should just set the `distribution` flag in `LoadNXSPE` and remove the instrument name checks.

Reply:
:+1: